fs/config/config.go

```diff
@@ -81,11 +81,11 @@ type BlobConfig struct {
 	FetchTimeoutSec      int64 `toml:"fetching_timeout_sec"`
 	ForceSingleRangeMode bool  `toml:"force_single_range_mode"`
 	MaxRetries           int   `toml:"max_retries"`
-	MinWaitMSec          int   `toml:"min_wait_msec"`
-	MaxWaitMSec          int   `toml:"max_wait_msec"`
+	MinWaitMsec          int   `toml:"min_wait_msec"`
+	MaxWaitMsec          int   `toml:"max_wait_msec"`
```

Comment:
go-staticcheck understands unit suffixes with a single leading capital letter, so it thinks "FooMSec" is "FooMsomething" with a unit of "seconds" instead of "Foo" with a unit of "milliseconds".